### PR TITLE
Add CODEOWNERS for Docker images.

### PR DESCRIPTION
**Pull request by PiotrSikora**
Signed-off-by: Piotr Sikora <piotrsikora@google.com>

### CODE REVIEW

**Review by howardjohn — APPROVED** [review]

**Comment by PiotrSikora** [comment]
Force-pushed to remove old `docker/OWNERS`, PTAL.

File: CODEOWNERS [hunk 1]
```diff
@@ -1,4 +1,5 @@
 *                                   @clarketm @fejta @Katharine @krzyzacy @michelle192837
+/docker/                            @howardjohn @sdake @geeknoid @duderino @linsun
```
Comment (fejta) [inline CODEOWNERS:2]: @clarketm do you want to be in here? `/hold cancel` if not
/cc @clarketm

**Review by fejta — APPROVED** [review]

File: CODEOWNERS (hunk 1 above)
Comment (clarketm) [reply CODEOWNERS:2]: I can PR and add myself in the future if needed.

**Pull request merged**